packages/dev/docs/pages/releases/2023-10-06.mdx

```diff
@@ -0,0 +1,102 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we've extended support for links across all collection components! To streamline integration with client-side routers, we've introduced the RouterProvider component which will allow you to seamlessly incorproate client-site routers without the need to wrap a router Link component. We have also continued to make improvements and bug fixes to React Aria Components since its beta release. React Spectrum also has some new additons, including DropZone and isPending prop to Button. 
```

Comment:
```suggestion
description: In this release, we've extended link support across all collection components! To streamline integration with client-side routers, we've introduced the RouterProvider component, allowing you to seamlessly incorporate client-side routers without wrapping a router Link component. Since its beta release, we have made improvements and bug fixes to React Aria Components. React Spectrum has new additions, including DropZone and isPending prop to Button. 
```

> allowing you to seamlessly incorporate client-side routers without wrapping a router Link component

This sentence needs a bit more work; whose "router Link" are we talking about?